2.4.41/Dockerfile

```diff
@@ -27,8 +27,7 @@ COPY contrib/*.schema /usr/local/etc/openldap/
 COPY contrib/DB_CONFIG /usr/local/etc/openldap/
 
 # Install OpenLDAP Server, give it permissionst to bind to low ports
-RUN yum install -y openldap openldap-servers openldap-clients && \
-    yum clean all -y && \
+RUN yum clean all -y && yum install -y openssl openldap openldap-servers openldap-clients procps-ng && \
```

Comment:
Don't you want the clean after the install to remove the extra nonsense in the cache?

Reply:
Ah, good point I thought the clean all at the end was a mistake as I'm used to seeing it ahead of the install command.